### ISSUE TITLE
Dark Mode support

### ISSUE THREAD

**Issue by aquarion** [tomtaylor/satellite-eyes#71]
Enhancement request: Ability to select different a different tile set if the OS is in dark mode.

**Comment by arlo-j** [comment]
Agreed!  Would love this, basically having Satellite Eyes adopt the dynamic desktop approach.

**Comment by hannesgassert** [comment]
Also, the Manage Map Styles window is broken in dark mode:
<img width="781" alt="Screen Shot 2021-05-25 at 14 07 20" src="https://user-images.githubusercontent.com/780100/119495266-87ef8c80-bd62-11eb-9b1d-6603d0328d42.png">

**Comment by tomtaylor** [comment]
This has landed on main/rc-1 and will be fixed in the next release.
